lib/rules/jsx-no-script-url.js

```diff
@@ -77,16 +75,17 @@ module.exports = {
         },
         required: ['name', 'props'],
         additionalProperties: false,
+        deprecated: true, // ?
```

Comment:
sounds good to me. does eslint actually check this?

Reply:
[yes](https://eslint.org/docs/latest/extend/custom-rules#rule-structure)